plugins/weather/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/weather/WeatherSpeechCommand.java

```diff
@@ -266,6 +267,17 @@ public String rainCheck(Map<String, EntityData> entities) {
 
 		return "i don't understand the date";
 	}
+	
+	private String stringifyRainCheck(double probability, String precipType, String suffix) {
+		final String chanceOf = "% chance of ";
+
+		if(round(probability) == 0) {
+			return "0% chance of rain " + suffix + "!";
```

Comment:
I think at 0% probability, the DS API returns "no-data" as the precip type. Maybe respond with "No precipitation today/tomorrow" instead